<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PeptML</class>
 <widget class="QDialog" name="PeptML">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pept-ML</string>
  </property>
  <widget class="QPushButton" name="ok_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="true_fraction">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>True Fraction</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkError">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate Error</string>
   </property>
  </widget>
  <widget class="QPushButton" name="help_button">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>help</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
